pkgs/development/web/postman/default.nix

```
@@ -1,6 +1,6 @@
 { stdenv, fetchurl, makeDesktopItem, wrapGAppsHook
 , atk, at-spi2-atk, alsaLib, cairo, cups, dbus, expat, gdk-pixbuf, glib, gtk3
```

```suggestion
, atk, at-spi2-atk, at-spi2-core, alsaLib, cairo, cups, dbus, expat, gdk-pixbuf, glib, gtk3
```

It looks like `at-spi2-core` is now required

---

I agree!